app/formats/document_types.yml

```diff
@@ -1,102 +1,83 @@
 ---
 - id: press_release
```

Comment:
The extraction of copy from `document_types.yml` is not an improvement I think. Everything used to be in one file, now it's spread out. If we want to keep things together that change together, than this would be better. Can we do this without putting it into a separate file?

Reply:
I agree with @tijmenb  - even if it means we're not able to load these strings using the `t()` function I think it's fine to have the copy in `document_types.yml`

Reply:
I've moved the format-specific text back into supertypes.yml and document_types.yml. Doing it this way requires some additional modelling, so we've now got a Guidance model and a lookup method to avoid too many [][]'s.